tests/auto/aggregation/testsyncadapter.cpp

```diff
@@ -70,6 +71,11 @@ void TestSyncAdapter::addRemoteContact(const QString &accountId, const QString &
 
     QContactPhoneNumber ncp;
     ncp.setNumber(phone);
+    if (mod == TestSyncAdapter::ExplicitlyModifiable) {
+        ncp.setValue(QContactDetail__FieldModifiable, true);
+    } else if (mod == TestSyncAdapter::ExplicitlyNonModifiable) {
+        ncp.setValue(QContactDetail__FieldModifiable, false);
+    }
```

Comment:
the "no modifiability flag value present" is difficult, because for a constituent contact it means "yes, modifiable" but for the aggregate it means "not modifiable... because I'm an aggregate".

So, the "ImplicitlyModifiable" value represents the POV of the constituent, rather than the aggregate.  That gets a bit confusing later on in the test when I test that the aggregate is ImplicitlyModifiable (ie, that it has no modifiability flag set)...